Resources/Prototypes/Catalog/Fills/Items/toolboxes.yml

```diff
@@ -69,14 +69,8 @@
   components:
   - type: StorageFill
     contents:
-      - id: Screwdriver
-      - id: Wrench
-      - id: Welder
-      - id: Crowbar
-      - id: Multitool
-      - id: Wirecutter
+      - id: ClothingBeltUtilityFilled
```

Comment:
if you are gonna include a belt raise the TC price to like 4 (also make sure everything fits inside the toolbox)

Reply:
the holster used to essentially be a valid utility belt that couldnt hold multitool
should still be 4 tc for other reasons tho

and yes it fits, see screenshot